### PR TITLE
Firebase

### PR DESCRIPTION
**Pull request by poornima-alugubelly**
Added Authentication using firebase

### CODE REVIEW

**Review by pritam-kr — APPROVED** [review]
please give us an overview of Firebase,.

Good to merge

File: src/components/Loader/Loader.tsx [hunk 1]
```diff
@@ -0,0 +1,4 @@
+import "./Loader.css";
+export const Loader: React.FC = () => {
```
Comment (msdeshmukh009) [inline src/components/Loader/Loader.tsx:2]: No need to write type here I guess.

File: src/types/index.ts [hunk 2]
```diff
@@ -3,3 +3,4 @@ export type { quizContext } from "./quiz.types";
 export type { quizReducerState } from "./quiz.types";
 export type { quizAction } from "./quiz.types";
 export type { quizOption } from "./quiz.types";
```
Comment (gopalbbharadva) [inline src/types/index.ts:5]: Can it be exported in single line as three of are coming from same files?

Comment (gopalbbharadva) [inline src/types/index.ts:5]: Can it be exported in single line as they are in same file.?

**Review by gopalbbharadva — APPROVED** [review]
Code looks fine and well structured. Functionality working fine.

**Review by msdeshmukh009 — APPROVED** [review]
The code looks clean. Great work !!

**Pull request merged**